WcaOnRails/spec/models/competition_wcif_spec.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+ONE_MINUTE_IN_CENTISECONDS = 60*100
```

Comment:
Ok, that worked well. I felt more comfortable with `in_centiseconds` though, which might drive you crazy, sorry!